sdk/src/wallet/core/operations/stronghold_backup/mod.rs

```diff
@@ -36,7 +34,7 @@ impl Wallet {
             // Backup with existing stronghold
             SecretManager::Stronghold(stronghold) => {
                 stronghold.set_password(stronghold_password).await?;
-                self.store_data_to_stronghold(stronghold).await?;
+                self.store_wallet_data_to_stronghold(stronghold).await?;
```

Comment:
Revert, we're in a Wallet impl block no need to put wallet in the name.

Reply:
Done